test/Utils/AssertExtensions.ts

```diff
@@ -25,6 +25,11 @@ export class JavaAssertionBuilder {
         return this;
     }
 
+    public isNotStrictEqualTo(val) {

```

Comment:
I guess in java it's `isNotEqualTo()` (without the strict). It might be easier to keep it like this to copy assertion code from java more easily.

Reply:
you are right!